src/ResourceManager/Cdn/Commands.Cdn/help/Get-AzureRmCdnEndpointNameAvailability.md

```diff
@@ -65,14 +65,10 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## INPUTS
 
-### None
```

Comment:
@prakharsharma10 please revert the change made here

Reply:
This file is auto-generated using the platyPS module